en/orm/associations.rst

```diff
@@ -721,6 +721,26 @@ and other sub-statements, will not be applied to the root query. Associations
 that are *not* loaded through joins (hasMany/belongsToMany), do not have the
 above restrictions and can also use result formatters or map/reduce functions.
 
+Association Conventions
+=======================
+
+By default, associations are expected to be set up and used in CamelCase style.
+Then chaining here to get a related table to work with is easy::
```

Comment:
```suggestion
By default, associations are should be configured and referenced using the CamelCase style.
This enables property chains to related tables in the following way::
```

Reworded a bit, we should avoid 'easy' as all things are easy when you know them :)